.github/workflows/ci.yml

```diff
@@ -880,6 +880,7 @@ jobs:
             - suite-delta-lake-databricks143
             - suite-delta-lake-databricks154
```

Comment:
Please take a look at `pt (suite-apache-ranger)` job logs. I see there:
```
presto-master       | 2024-11-05T00:55:12.761+0545	ERROR	main	org.apache.ranger.plugin.util.PolicyRefresher	PolicyRefresher(serviceName=dev_trino): failed to refresh policies. Will continue to use last known version of policies (-1)
presto-master       | com.sun.jersey.api.client.ClientHandlerException: java.net.UnknownHostException: 
```

Can we configure Ranger in a way that does not throw such exceptions?

Reply:
No. There is no configuration to disable this exception.